addons/binding/org.openhab.binding.airquality/.gitignore

```diff
@@ -0,0 +1,5 @@
+.classpath
```

Comment:
Please remove binding-specific gitignore file from the PR. We should depend on the overall .gitignore file in the root of the project ( @kaikreuzer : I know we are using eclipse, but could we add the .idea and .*.iml for IntelliJ anyway?)